ui.apps/src/main/content/jcr_root/apps/aemtraining/components/structure/content/body.html

```diff
@@ -0,0 +1,19 @@
+<!-- Page Wrapper -->
```

Comment:
Please always use "HTL-style" comments in HTL files.
An HTL-style comment goes like this `<!--/* comment comment comment */-->`